frontend/model/notifications/templates.js

```diff
@@ -409,19 +410,22 @@ export default ({
       }]
     }
   },
-  PAYMENT_RECEIVED (data: { groupID: string, creatorID: string, amount: string, paymentHash: string }) {
+  PAYMENT_RECEIVED (data: { groupID: string, creatorID: string, amount: number, currency: string, paymentHash: string }) {
```

Comment:
This is now receiving a new `currency` parameter. However, it doesn't seem to be used, and even if it were, per the recent docs, it needs to be very carefully managed as the old contracts will not pass it in.

If it's not used however, it should just be removed (and likewise `group.js` updated to remove it too)